_includes/search.html

```diff
@@ -1,23 +1,44 @@
 <!-- HTML elements for search -->
+
+<!-- This is installed from the https://github.com/slashdotdash/jekyll-lunr-js-search Gem -->
+<script src="/js/search.min.js" type="text/javascript" charset="utf-8"></script>
+
 <form id="search-form">
```

Comment:
The `form` is currently making it so that hitting enter refreshes the page because the form gets submitted. I'd just remove it, since that's probably not what you want.

Reply:
Another way about this if you don't wanna fiddle with CSS after you remove the form is to add this attribute to it... :grimacing: 

```
action="javascript:void(0);"
```

Reply:
done!